pkgs/development/python-modules/pcbnew-transition/default.nix

```diff
@@ -6,14 +6,15 @@
 , versioneer
 }:
 buildPythonPackage rec {
-  pname = "pcbnewTransition";
+  pname = "pcbnew-transition";
```

Comment:
Is this really the correct normalization? Using the `normalize` function from https://packaging.python.org/en/latest/specifications/name-normalization/#name-normalization, I get a different result:

```
$ python
...
>>> import re
>>> def normalize(name):
...     return re.sub(r"[-_.]+", "-", name).lower()
...
>>> normalize("pcbnewTransition")
'pcbnewtransition'
```

Reply:
you are correct. I will rename it.